.travis.yml

```diff
@@ -64,8 +64,7 @@ script:
                osx)
                  pushd tests &&
                  cargo check &&
-                 cargo check --features=all &&
-                 cargo test tree --release
+                 cargo test tree
```

Comment:
Did you remove `--all-features` deliberately or did you forget to add it?

Reply:
deliberate, and I got rid of all of the features on the tests subcrate